src/System.CommandLine.Tests/ResponseFileTests.cs

```diff
@@ -336,5 +336,31 @@ public void Response_files_can_refer_to_other_response_files()
             result.FindResultFor(option3).GetValueOrDefault().Should().Be(3);
             result.Errors.Should().BeEmpty();
         }
+
+        [Fact]
+        public void When_response_file_options_or_arguments_contain_trailing_spaces_they_are_ignored()
```

Comment:
This looks like three separate tests and breaking them up will make it a little clearer.

Trailing spaces for options:
```csharp
var responseFile1 = ResponseFile("--option1 ", "value1 ", "--option2 ", "2 ");
```

Leading spaces for options:
```csharp
var responseFile2 = ResponseFile(" --option1", " value1", " --option2", " 2");
```

Trailing *and* leading spaces for arguments:
```csharp
var responseFile3 = ResponseFile(" --option1 ", " value1 ", " --option2 ", " 2 ");
```

Reply:
Agreed and done. Wasn't sure how granular the tests should be, but I realised after committing that the name was wrong anyway.

Reply:
Nice to add in tabs as well!